util/format-csv-record.js

```diff
@@ -0,0 +1,68 @@
+// Utility functions for formatting the csv records
+// Constants
+var QUALITY_ID_TO_MEASURE_MAP = require('./constants/quality-id-to-measure-map');
+// Utils
+var formatSubmissionMethod  = require('./format-submission-method');
+var formatDecileGenerator   = require('./format-decile-generator');
+/**
+ *
+ * @param {{
+ *  measureName: string,
+ *  qualityId: string,
+ *  submissionMethod: string,
+ *  measureType: string,
+ *  benchmark: string,
+ *  decile1: string?,
+ *  decile2: string?,
+ *  decile3: string?,
+ *  decile4: string?,
+ *  decile5: string?,
+ *  decile6: string?,
+ *  decile7: string?,
+ *  decile8: string?,
+ *  decile9: string?,
+ *  decile10: string?,
+ *  isToppedOut: string
+ *  }} record - csv record object
+ * @param {{
+ *  benchmarkYear: string,
+ *  performanceYear: string
+ * }} options - 4 digit year strings for benchmark and performance years
+ * @returns {{
+ *  measureId: string,
+ *  benchmarkYear: string,
+ *  performanceYear: string,
+ *  submissionMethod: string,
+ *  deciles: Array} | undefined} - benchmark object
+ */
+var formatCsvRecord = function formatCsvRecord(record, options) {
+  /**
+   * NOTE: Some of the benchmarks don't correspond to
+   * any of the measures currently in our json.
+   */
+  var measure = QUALITY_ID_TO_MEASURE_MAP[record.qualityId] || {};
```

Comment:
So if a measure doesn't exist for the benchmark it's still being added will a null measureId? Don't think that's the behavior we want. I'd say don't worry that we have benchmarks that don't correspond to measures - the measure could be deprecated or it may be a non-QPP measure (which hasn't been finalized yet).